File: 2011/s2.py
# code by savir singh

def check_work(list1):
    global answers
    global correct
    for k in range(len(list1)):
        if list1[k] == answers[k]:
            correct += 1

n = int(input())
choices = []
answers = []
correct = 0

for i in range(n):
    r = input()
    choices.append(r)

for j in range(n):
    r = input()
    answers.append(r)

check_work(choices)

print(correct)

File: 2011/s2.cpp
// code by savir singh

#include <bits/stdc++.h>
#define ll long long
#define del erase
using namespace std;

int main()
{
    ios_base::sync_with_stdio(false);
    cin.tie(NULL);
    ll n;
    cin >> n;
    vector<string> stud (n);
    vector<string> ans (n);
    ll correct=0;
    for (ll i=0; i<n; i++) {
        cin >> stud[i];
    }
    for (ll i=0; i<n; i++) {
        cin >> ans[i];
    }
    for (ll j=0; j<n; j++) {
        if (stud[j]==ans[j]) {
            correct++;
        }
    }
    cout << correct;
}
